docs/profiling.rst

```diff
@@ -1,6 +1,12 @@
+.. TODO: This section would be removed after we migrate to www_rbac completely.
```

Comment:
This line will not be compiled into the documentation.
Ref: http://docutils.sourceforge.net/docs/user/rst/quickref.html#comments